src/main/groovy/jp/co/soramitsu/devops/docker/DockerPlugin.groovy

```diff
@@ -142,7 +143,25 @@ class DockerPlugin implements Plugin<Project> {
         }
     }
 
-    static void setupDockerfileCreateTask(Project project, @NonNull File jar) {
+    static void setupDockerCopyCustomFilesTask(Project project, DockerConfig dockerConfig) {
+        def customFiles = dockerConfig.customFiles
+        if (customFiles == null || customFiles.isEmpty()) {
+            println(format("Task ${SoraTask.dockerCopyFiles} has been omitted since no files were supplied."))
+            return
+        }
+        project.tasks.register(SoraTask.dockerCopyFiles, Dockerfile).configure { Dockerfile t ->
+            t.group = DOCKER_TASK_GROUP
+            t.description = "Copy custom files to ${getDockerContextDir(project).path}"
+            t.dependsOn(SoraTask.dockerfileCreate)
+
+            customFiles.each { source, destination ->
+                println(format("File mapping: ${source} -> /${destination}"))
+                t.addFile(source, "/$destination")
```

Comment:
also - `destination` should be inside docker context dir (e.g. build/docker, or getDockerContextDir(project))